webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -78,8 +81,21 @@ export default class Logs extends React.PureComponent<Props, State> {
         }
     }
 
+    toggleMode = () => {
+        this.setState({
+            ...this.state,
```

Comment:
why is updating whole state needed?

Reply:
Updated now. Could you please review it again?